src/libraries/System.Text.Json/tests/System.Text.Json.Tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Xunit;
+
+[assembly: ActiveIssue("Interpreter with debug runtime can be very slow", typeof(PlatformDetection), nameof(PlatformDetection.IsMonoInterpreter), nameof(PlatformDetection.IsDebugRuntime))]
```

Comment:
@akoeplinger Not sure if we have a better flag here. I think having a tracking issue is redundant since it is not something worth addressing in my opinion.

Reply:
Yeah I checked and we don't have a better way right now since all of the `Conditional*` attributes can't be applied to assembly. I'm fine with this.